test/unit/example.txt

```diff
@@ -0,0 +1,5 @@
+describe('{{name}}', () => {
+  it('should work', async () => {
+    console.log('hello world')
```

Comment:
```suggestion
    const result = 'success'
    expect(result).toBe('success')
```

Should this have at least one assertion?

Reply:
Updated in https://github.com/vercel/next.js/pull/33001/commits/e338dbf409b32f79d76d75d7dd5a93159d8d251c